README.md

```diff
@@ -239,7 +227,6 @@ Use bower `bower install globalize`, or npm `npm install globalize`.
 1. `git clone https://github.com/jquery/globalize.git`.
 1. [Build the distribution files](https://github.com/jquery/globalize/#build).
```

Comment:
This was previously wrong. Please, can you fix it? `https://github.com/jquery/globalize/#build` to `#build`.